README.md

```diff
@@ -36,37 +36,41 @@ All ZLC meeting are posted on the [Zowe Development Calendar](https://lists.open
 
 # Zowe sub-projects
 
-The following are the current sub-projects that are part Zowe by maturity level
+The following are the current sub-projects that are part of Zowe by maturity level
 
 ## Incubation
 
+* *None*
 
 ## Active
 
-* Zowe API Mediation layer
-* Zowe CLI
-* Zowe Web Desktop
-* Zowe Onboarding
-* Zowe CI/CD
+* [Zowe API Mediation layer](https://github.com/zowe/community#zowe-api-mediation-layer)
+* [Zowe CLI](https://github.com/zowe/community#zowe-cli)
+* [Zowe App Framework](https://github.com/zowe/community#zowe-app-framework)
 
 ## Emeritus
 
+* *None*
+
+There are other efforts that sit outside the project structure that focus on cross sub-project coordination. Learn more about the [Zowe Community Teams](https://github.com/zowe/community#zowe-community-teams).
```

Comment:
Because this is put below emeritus instead of active, it may be confusing as to the status. Maybe not, just a thought though.

Reply:
Thanks - moved it up to avoid that confusion.